### PR TITLE
implement navbar menu (closes #35)

### LINKED ISSUE #35: Fix mobile
We've received a couple of complains about mobile not having a "Menu" button. We used to have one, and we could probably just revive that code.

### PR DESCRIPTION
**Pull request by akisblack**
the code is shit

### CODE REVIEW

**Review by openssl-rand — APPROVED** [review]
If it works just merge it

File: src/lib/Nav.svelte [hunk 1]
```diff
@@ -10,6 +14,13 @@
 		>
 	</a>
 
+	<input type="checkbox" id="toggle-menu" />
+	<label class="menu-icon" for="toggle-menu">
+		<a id="menu-icon">
+			<IconBars />
+		</a>
+	</label>
+
```
Comment (Midou36O) [inline src/lib/Nav.svelte:23]: Could you explain what this do?

**Review by Midou36O — COMMENTED** [review]
I've checked the files real quick:

File: src/lib/Nav.svelte (hunk 1 above)
Comment (akisblack) [reply src/lib/Nav.svelte:23]: it uses a checkbox instead of javascript onclick to trigger the opening of the menu, and thus works completely without js

Comment (Midou36O) [reply src/lib/Nav.svelte:23]: :+1:

**Review by Midou36O — APPROVED** [review]

**Review by Odyssey346 — APPROVED** [review]
if the code "sucks", it doesn't matter unless it actually slows the user down. if it works its good also i guess the svelte compiler doesn't care idk but i'll approve

**Pull request merged**